frontend/pages/[resourceType]/search.tsx

```diff
@@ -0,0 +1,40 @@
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next';
+import { ArtifactResourceType } from '@/util/types/fhir';
+import { Divider, Group, Stack, Text } from '@mantine/core';
+import BackButton from '@/components/BackButton';
+import SearchComponent from '@/components/SearchComponent';
+
+export default function ResourceSearchPage({ resourceType }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  return (
+    <div>
+      <Stack spacing="xs">
+        <div>
+          <Group>
+            <BackButton />
+            <Text size="xl" weight={700} color="gray">
+              {resourceType} Search
+            </Text>
+          </Group>
+        </div>
+        <Divider my="sm" style={{ paddingBottom: '6px' }} />
+        <SearchComponent resourceType={resourceType} />
+      </Stack>
+    </div>
+  );
+}
+
+export const getServerSideProps: GetServerSideProps<{
```

Comment:
Does this need to be a `getServerSideProps` at all? That should be reserved for when we are doing something that requires backend communication. It looks like this is just checking request.query for a resource type, so it might make more sense to just do that on the client and render an error somewhere if it happens? If a user goes to `/Patient/search` or something then shame on them lol